src/common/com/intellij/plugins/haxe/ide/generation/CreateGetterSetterFix.java

```diff
@@ -46,7 +42,7 @@ boolean accept(String name, Set<String> names) {
     }, GETTERSETTER {
       @Override
       boolean accept(String name, Set<String> names) {
-        return !names.contains(HaxePresentableUtil.getterName(name)) &&
+        return !names.contains(HaxePresentableUtil.getterName(name)) ||
```

Comment:
&& operator was causing other (either getter or setter) not being generated when only one of them (either getter or setter) is already present